docs/tasks/debug-application-cluster/audit.md

```diff
@@ -284,6 +284,129 @@ Events are POSTed as a JSON serialized `EventList`. An example payload:
 }
 ```
 
+### Log Collector Examples
+
+#### Use fluentd to collect and distribution audit events from log file
+
+[Fluentd][fluentd] is an open source data collector for unified logging layer.
+In this example, we will use fluentd to split audit events by different namespaces.
+
+1. install [fluentd, fluent-plugin-forest and fluent-plugin-rewrite-tag-filter][fluentd_install_doc] in the kube-apiserver node
+2. create a config file for fluentd
+```
+$ cat <<EOF > /etc/fluentd/config
+<source>
+    @type tail
+    # audit log path of kube-apiserver
+    path /var/log/kube-audit
+    pos_file /var/log/audit.pos
+    format /^(?<time>[^\]]*) AUDIT: id=\"(?<id>[^ ]*)\" stage=\"(?<stage>[^ ]*)\" ip=\"(?<ip>[^ ]*)\" method=\"(?<method>[^ ]*)\" user=\"(?<user>[^ ]*)\" groups=\"(?<groups>[^ ]*)\" as=\"(?<as>[^ ]*)\" asgroups=\"(?<asgroups>[^ ]*)\" namespace=\"(?<namespace>[^ ]*)\" uri=\"(?<uri>[^ ]*)\" response=\"(?<response>[^ ]*)\"$/
```

Comment:
+1 for json format, it would also make e2e testing easier. I meant to add this before 1.7, but didn't get to it. I agree that we should just dump the audit.Event (versioned) into the log, with 1 event per line.